src/interfaces/IContractData.ts

```diff
@@ -1,14 +1,22 @@
+/**
+ * Represents the information for a smart contract.
+ * @remarks
+ * This interface is used to track the smart contract information, including the
+ * storage fee, maximum gas, contract data (as text or bytecode), contract address,
+ * and an optional datastore for the smart contract's operational storage data.
+ *
+ * @see fee of type `bigint` represents the storage fee for the smart contract.
+ * @see maxGas of type `bigint` represents the maximum gas that can be consumed by the smart contract.
```

Comment:
is this the max gas fees that can be consumed during the deployment or while calling it ?